conanfile.py

```diff
@@ -160,13 +165,38 @@ def build(self):
         cmake.configure()
         cmake.build()
 
+        if self.options.get_safe("enable_sentry", False):
+            # Upload debug symbols to sentry
+            sentry_project = self.conf.get("user.curaengine:sentry_project", "", check_type=str)
+            sentry_org = self.conf.get("user.curaengine:sentry_org", "", check_type=str)
+            if sentry_project == "" or sentry_org == "":
+                raise ConanInvalidConfiguration("sentry_project or sentry_org is not set")
+            if which("sentry-cli") is None:
+                self.output.warn("sentry-cli is not installed, skipping uploading debug symbols")
+                self.output.warn("sentry-cli is not installed, skipping release creation")
+                return
```

Comment:
You should not `return` here because it would skip some code potentially added after the sentry-uploading step (even though this case should not happen)

Reply:
bit only for needed for setting up the sentry release and bookkeeping in our Sentry server it self. having it raise an error will only force developers pr other build systems to have that sentry-cli installed.

Which turns out to be rather annoying on a dev setup

Reply:
I agree that this should not raise an error, but the following part should be under an `else` statement. This way, if you later add more code to the `validate` method (not related to sentry), it will always be executed.

Reply:
That would yet again add an extra indent for a huge chunk of code which I personally dislike. Also I think anyone working with python should be able to read the code how it is now.

But if you feel strongly about this feel free to change it

Reply:
:+1: